symengine/cwrapper.h

```diff
@@ -100,6 +101,37 @@ int is_a_Rational(const basic s);
 //! Return 1 if s is an Symbol, 0 if not.
 int is_a_Symbol(const basic s);
 
+
+//! Wrapper for std::vector<int>
+
+typedef struct CVectorInt CVectorInt;
+
+CVectorInt* vectorint_new();
+
+// 'self' must point to allocated memory of size 'size'. The function returns 0
+// if std::vector<int> can be initialized using placement new into 'self',
+// otherwise 1 if 'size' is too small or 2 if 'self' is not properly aligned.
+// No memory is leaked either way.
```

Comment:
Can we `enum` our error_types, like it is mentioned [here](http://stackoverflow.com/questions/1102542/how-to-define-an-enumerated-type-enum-in-c#1102556)? However, there is this issue that I got to know from one of the comments.
> @InnaKamoze isn't enum defined in C89? So unless you're not using ANSI C, I think that enum is supported.